proto/kvrpcpb.proto

```diff
@@ -241,6 +258,32 @@ message BatchRollbackResponse {
     KeyError error = 2;
 }
 
+// CheckTxnStatusRequest checks the status of a transaction.
+// If the TTL of the transaction is exhausted, abort that transaction.
+// Otherwise, returns the TTL information in addition to commit/rollback.
+// CheckTxnStatusRequest also pushes the minCommitTS forword for large transaction.
+//
+// In the future, TiDB may not store TTL in the secondary lock, and it calls this API to
+// get the TTL from the primary key.
```

Comment:
May this behavior reduces the performance, when we can be sure that the lock is not expired? I think maybe it's better to keep a TTL in the secondary keys, and if secondary's TTL is expired, check the primary key's TTL instead.

Reply:
Keep the TTL on the secondary keys is also acceptable.
This change is not the final version.